tower_cli/cli/action.py

```diff
@@ -41,3 +44,40 @@ def parse_args(self, ctx, args):
             click.echo(ctx.get_help())
             ctx.exit()
         return super(ActionSubcommand, self).parse_args(ctx, args)
+
+    def format_options(self, ctx, formatter):
```

Comment:
So I take it that this is called by the click library? A tiny comment noting that might be nice.